### ISSUE TITLE
No emails being sent

### ISSUE THREAD

**Issue by clayy503** [CosmodiumCS/DucKey-Logger#2]
I followed the Youtube video as well as the readme instructions and everything seems to work except I am not getting any emails. I see the c.cmd in the startup folder and the ducky runs the PowerShell script same as in the video (red permissions errors and all). Not sure where I am going wrong?

**Comment by PrettyBoyCosmo** [comment]
Hey there could be a few issues you may have but be assured the payload does work.
- first, please make sure you watched the updated [tutorial](https://www.youtube.com/watch?v=uHIZZYFeVJA) and include all of the files in your micro sd card
- second make sure you are using a gmail account and it has LSA Access enabled
- third, make sure you have the correct credentials stored in double-quotes in the p.ps1 file

**Comment by clayy503** [comment]
Hi CosmodiumCS, I tried your suggestions 3x over, and still nothing. No email is sent. I looked through all the files you have in the repo, and Idk what is going wrong. Seems like every part of the program is working except for the smtp.send. I did not put the readme and the license files on the ducky.

- first, please make sure you watched the updated tutorial and include all of the files in your micro sd card 
- second make sure you are using a gmail account and it has LSA Access enabled - I checked this.
- third, make sure you have the correct credentials stored in double-quotes in the p.ps1 file - I checked this.

**Comment by PrettyBoyCosmo** [comment]
the only issues you could be having is that LSA is not enabled or your windows account does not have the permissions to send emails via smtp. if you want i can help you sometime tomorrow in more detail via our [discord](https://discord.gg/E5YBNpv8hS).
just dm me

**Comment by clayy503** [comment]
Thanks, I might take you up on that offer. I'll dig more into my windows account not having proper permissions to send emails via smtp, although I was able to send via smtp with other scripts using smtp.gmail.com and the same email/password used here. I also tried from work and home so 2 different environments. LSA is enabled from what I can tell. If that link you show is the only way to enable it, it is definitely enabled. Like I said in my previous comments, It seems the smtp piece is the only thing not working.
